.buildkite/basic/react-native-android-full-pipeline.yml

```diff
@@ -102,6 +102,7 @@ steps:
           - "0.74"
           - "0.76"
           - "0.77"
+          - "0.78"
```

Comment:
should we drop 0.72 from the matrix yet? I can't remember what the decision was on the support matrix, but 0.76 is now unsupported by react native.

Reply:
I don't think we can drop 0.72 - our test matrix policy is the latest 3 versions, plus even versions back to 0.72